packages/wrangler/src/init.ts

```diff
@@ -588,19 +617,12 @@ export async function initHandler(args: InitArgs) {
 						readFileSync(path.join(getBasePath(), `templates/${template}`))
 					);
 
-					logger.log(
```

Comment:
doesn't removing this line mean you don't get feedback after the `src/index.js` gets created?

Reply:
I moved it down, I can move it back.